### PR TITLE
Rewrite some ES6 code in an ES5-compatible way

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: app/assets/javascripts/blacklight/blacklight.js [hunk 1]
```diff
@@ -37,7 +37,9 @@ Blacklight.listeners().forEach(function (listener) {
   });
 });
 Blacklight.onLoad(function () {
-  const elem = document.querySelector('.no-js');
+  const elem = document.querySelector('.no-js'); // The "no-js" class may already have been removed because this function is
```
Comment (jcoyne) [inline app/assets/javascripts/blacklight/blacklight.js:40]: Is const in es5?

File: app/javascript/blacklight/search_context.js [hunk 2]
```diff
@@ -15,8 +15,8 @@ Blacklight.doSearchContextBehavior = function() {
   })
 };
 
-Blacklight.csrfToken = () => document.querySelector('meta[name=csrf-token]')?.content
-Blacklight.csrfParam = () => document.querySelector('meta[name=csrf-param]')?.content
+Blacklight.csrfToken = function() { return (document.querySelector('meta[name=csrf-token]') || {}).content }
+Blacklight.csrfParam = function() { return (document.querySelector('meta[name=csrf-param]') || {}).content }
```
Comment (jrochkind) [inline app/javascript/blacklight/search_context.js:19]: Now I wonder why Blacklight.csrfToken/csrfParam are defined twice, in two different places, identically? But that pre-dated this PR, so not relevant to approving it, just wondering.

Comment (jcoyne) [reply app/javascript/blacklight/search_context.js:19]: One is the compiled distributed code and one is the source code.

**Comment by jcoyne** [comment]
I'd prefer #2510 as an alternative as it makes all of BL ES2015 compilant.

**Comment by jrochkind** [comment]
@cbeer I think maybe there is in fact no way to load the Javascript individually via sprockets even before this PR? That went away when the new system of supporting dual sprockets/npm was created?

I don't think I've been able to do it since then. I do miss it. I wish I could do it. It was a problem before this PR. This PR may not make it easier to step out of it, if it's desired to step out of it -- you saying something about it makes me think for the first time it was a bug/error/confusion on my part that i wasn't able to load the JS individually via sprockets, instead of an intended design? I always figured before now it was known and intended that this wasn't something Blacklight supported anymore.

**Comment by cbeer** [comment]
I don't know anything about that; I'd suggest leaving your comments over on the relevant PR https://github.com/projectblacklight/blacklight/pull/1727?

**Comment by jrochkind** [comment]
I also don't know anything about it, or know if anyone does?

I was replying to your comment here:

> And the thought is... if you're loading the javascript individually, you probably have your own javascript preprocessor if you care about IE11? I guess I can buy that.

**Pull request closed without merging**